### PR TITLE
docs(samples): add feature importance to predict sample

### PR DESCRIPTION
**Pull request by leahecole**
keeping do not merge on here until the canonical Python is merged

### CODE REVIEW

**Comment by leahecole** [comment]
@JustinBeckwith - linkinator extraordinaire, I'm having trouble figuring out from these logs exactly what link is broken - any ideas and or resources for a linkinator newb?

**Comment by leahecole** [comment]
Ugh I think code coverage might be looking at the generated docs - it was passing before and the only change I made was to skip that link @JustinBeckwith 🤷‍♀

**Comment by JustinBeckwith** [comment]
Uhhh @bcoe?

File: linkinator.config.json [hunk 1]
```diff
@@ -3,6 +3,7 @@
   "skip": [
     "https://codecov.io/gh/googleapis/",
     "www.googleapis.com",
-    "setEndpoint.js"
+    "setEndpoint.js",
+    "https://www.tensorflow.org/js"
```
Comment (bcoe) [inline linkinator.config.json]: I'm a little surprised we needed to add this, isn't the tensorflow.org website available?

File: samples/tables/predict.v1beta1.js [hunk 2]
```diff
@@ -63,14 +63,36 @@ async function main(
...
           console.log(`Prediction results:`);
 
           for (const result of responses[0].payload) {
             console.log(`Predicted class name: ${result.displayName}`);
             console.log(`Predicted class score: ${result.tables.score}`);
+
+            // Get features of top importance
+            const featureList = [];
+
+            for (const feature of result.tables.tablesModelColumnInfo) {
```
Comment (bcoe) [inline samples/tables/predict.v1beta1.js]: I might be tempted to do this in a single map:

```js
const featureList = result.tables.tablesModelColumnInfo.map((columnInfo) => {
  return {
    importance: feature.featureImportance,
    displayName: feature.columnDisplayName
  }
})
```

File: linkinator.config.json (hunk 1 above)
Comment (leahecole) [reply linkinator.config.json]: it is surprisingly flaky!

**Comment by leahecole** [comment]
working on the lint - I upgraded to catalina and changed my shell to zsh and am dealing with that whole headache as I work on this

**Pull request merged**